### DISCUSSION TITLE
使用地图模块在默认配置下无法开启四处看看功能

### DISCUSSION THREAD

**Discussion by li2282231** [NSRingo/.github#26] [Q&A 问答]
经过我的实测，使用地图模块在默认配置情况下，无法使用四处看看功能；例如搜索东京，拉近东京地图后无法显示四处看看的望远镜图标，具体表现是：Mac地图app上右上角的望远镜四处看看图标显示灰色、iOS地图app左下角没有望远镜四处看看的图标。
如果在boxjs中将动态配置改为美国，则可以正常显示出四处看看的望远镜图标，此时功能正常开启（但是这样国内地图就没有高德数据了）；这时如果将动态配置再改为中国大陆，可以短暂的使用四处看看，在冷启动地图app后四处看看又会再次无法使用，还请大佬核查，谢谢！
![eca990ba-0756-487e-8d2f-af78ff20cc85](https://github.com/user-attachments/assets/3173da4f-2b8e-40fc-8170-32ba9e676777)

**Comment by VirgilClyne** [comment]
https://github.com/orgs/NSRingo/discussions/25

**Reply by li2282231** [reply]
大佬，感谢回复，我用的是surge，手机和Mac都是surge，都用的同一个配置文件。并不是圈x，当前iOS版本是18.1.1、Mac版本是15.2，目前我测试不管是手机还是Mac都有这个问题？
